tron/utils/persistable.py

```diff
@@ -10,3 +10,8 @@ class Persistable(ABC):
     @abstractmethod
     def to_json(state_data: Dict[Any, Any]) -> Optional[str]:
         pass
+
+    @staticmethod
+    @abstractmethod
+    def from_json(state_data: str) -> Dict[Any, Any]:
+        pass
```

Comment:
I think I returned here Dict[Any, Any] cause this is what we were expecting to get in the ``to_json`` function

Reply:
whoops - missed responding this this one (although, we've since been talking about this in another thread): i think ideally that from_json would return the deserialized object - not the same intermediate representation that we used to serialize the initial object